src/dom/dom.js

```diff
@@ -1516,11 +1521,11 @@ p5.Element.prototype.removeClass = function(c) {
 
 /**
  *
- * Checks if specified class already set to element
+ * Checks if specified `class` is already set on the element.
  *
  * @method hasClass
- * @returns {boolean} a boolean value if element has specified class
- * @param c {String} class name of class to check
+ * @param {String} c The `class` name to check
```

Comment:
"class" don't need to be enclosed in code tag here, or the next one in the param tag below needs to be enclosed.